Source/WebCore/page/scrolling/AsyncScrollingCoordinator.h

```diff
@@ -76,6 +76,7 @@ class AsyncScrollingCoordinator : public ScrollingCoordinator {
     WEBCORE_EXPORT void setMouseIsOverContentArea(ScrollableArea&, bool) override;
     WEBCORE_EXPORT void setMouseMovedInContentArea(ScrollableArea&) override;
     WEBCORE_EXPORT void setLayerHostingContextIdentifierForFrameHostingNode(ScrollingNodeID, std::optional<LayerHostingContextIdentifier>) override;
+    LocalFrameView* frameViewForScrollingNode(LocalFrame& localMainFrame, ScrollingNodeID) const;
```

Comment:
This parameter name should be changed to rootFrame.  You might also want to add ASSERT(rootFrame.isRootFrame()) to make that more clear in the code and verify in debug builds.

Reply:
Fixed.